presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduDistributedQueries.java

```diff
@@ -148,6 +149,7 @@ public void testShowColumns()
     }
 
     @Override
+    @Test
```

Comment:
> Add missing @test annotation

squash the commit with the next one, since otherwise the annotation is not really missing here.

Reply:
too late